table/plain/plain_table_factory.cc

```diff
@@ -123,6 +123,26 @@ std::string PlainTableFactory::GetPrintableOptions() const {
   return ret;
 }
 
+Status PlainTableFactory::ValidateOptions(
+    const DBOptions& db_opts, const ColumnFamilyOptions& cf_opts) const {
+  if (cf_opts.comparator->CanKeysWithDifferentByteContentsBeEqual()) {
+    // More checks for incompatible hashing (see also
+    // ColumnFamilyData::ValidateOptions)
+    if (cf_opts.prefix_extractor) {
```

Comment:
Just me learning - why don't we do this check in `ColumnFamilyData::ValidateOptions()` like https://github.com/facebook/rocksdb/pull/10293/files?show-viewed-files=true&file-filters%5B%5D=#diff-de466ee8895040940921ff178dd5e6786ea79e954052f578c8444afd99ddf799R1434?

Reply:
IIRC setting a prefix extractor with plain table enabled an in memory bloom filter. Although odd, setting prefix extractor in general is not strictly incompatible.